src/utils.h

```diff
@@ -90,7 +93,7 @@ static inline void *alloc_simd(size_t n) {
 #if defined(_WIN32)
         void *p = _aligned_malloc(n, 16);
 #else
-        void *p = aligned_alloc(16, n);
+        void *p = memalign(16, n);
```

Comment:
man memalign says `memalign` is obsolete. Are you targeting an old compiling environment?

Reply:
compute.o:/home/mykaralw/WORK/soft/git/victorvde/jpeg2png/utils.h:93: more undefined references to `aligned_alloc' follow
collect2: error: ld returned 1 exit status

Debian Wheezy.